### PR TITLE
Improve user interface for creating new messages in messages plugin

### PR DESCRIPTION
**Pull request by paroga**

### CODE REVIEW

File: plugins/messages/app/models/message.rb [hunk 1]
```diff
@@ -25,13 +25,18 @@ class Message < ActiveRecord::Base
   validates_length_of :subject, :in => 1..255
   validates_inclusion_of :email_state, :in => EMAIL_STATE.values
 
+  after_initialize do
+    @send_method = 'recipients' unless @send_method
```
Comment (wvengen) [inline plugins/messages/app/models/message.rb]: `@send_method ||= 'recipients'`?

File: plugins/messages/app/models/message.rb [hunk 2]
```diff
@@ -40,14 +45,33 @@ def add_recipients(users)
...
+  end
+
+  def messagegroup_id=(messagegroup_id)
+    @group_id = messagegroup_id if send_method == 'messagegroup'
+  end
+
   def order_id=(order_id)
     @order_id = order_id
-    add_recipients Order.find(order_id).users_ordered unless order_id.blank?
+    @send_method = 'order' unless send_method
```
Comment (wvengen) [inline plugins/messages/app/models/message.rb]: also here, `@send_method ||= 'order'`

**Comment by paroga** [comment]
i've added the translations

File: plugins/messages/config/locales/en.yml [hunk 3]
```diff
@@ -3,11 +3,20 @@ en:
...
+        messagegroup_id: Messagegroup
+        order_id: Order
+        ordergroup_id: Ordergroup
         private: Private
-        recipient_tokens: Recipients
-        send_to_all: Send to all members
+        recipient_tokens: (Additional) recipients
+        send_method:
+          all: Send to all members
+          recipients: Send to sepcific members
```
Comment (wvengen) [inline plugins/messages/config/locales/en.yml]: _specific_

File: plugins/messages/config/locales/en.yml [hunk 4]
```diff
@@ -3,11 +3,20 @@ en:
...
+          order: Send to memebers, who participated at an order
```
Comment (wvengen) [inline plugins/messages/config/locales/en.yml]: _members_

**Pull request merged**